conformance/tests/conformance_test.go

```diff
@@ -37,52 +33,32 @@ import (
 
 func TestConformance(t *testing.T) {
 	g := NewWithT(t)
-	cfg, err := config.GetConfig()
-	g.Expect(err).To(BeNil())
-
-	client, err := client.New(cfg, client.Options{})
-	g.Expect(err).To(BeNil())
-
-	g.Expect(v1alpha2.AddToScheme(client.Scheme())).To(Succeed())
-	g.Expect(v1.AddToScheme(client.Scheme())).To(Succeed())
-	g.Expect(v1beta1.AddToScheme(client.Scheme())).To(Succeed())
-
-	supportedFeatures := suite.ParseSupportedFeatures(*flags.SupportedFeatures)
-	exemptFeatures := suite.ParseSupportedFeatures(*flags.ExemptFeatures)
 
 	t.Logf(`Running conformance tests with %s GatewayClass\n cleanup: %t\n`+
 		`debug: %t\n enable all features: %t \n supported features: [%v]\n exempt features: [%v]`,
 		*flags.GatewayClassName, *flags.CleanupBaseResources, *flags.ShowDebug,
 		*flags.EnableAllSupportedFeatures, *flags.SupportedFeatures, *flags.ExemptFeatures)
 
-	expSuite, err := suite.NewExperimentalConformanceTestSuite(suite.ExperimentalConformanceOptions{
-		Options: suite.Options{
-			Client:                     client,
-			GatewayClassName:           *flags.GatewayClassName,
-			Debug:                      *flags.ShowDebug,
-			CleanupBaseResources:       *flags.CleanupBaseResources,
-			SupportedFeatures:          supportedFeatures,
-			ExemptFeatures:             exemptFeatures,
-			EnableAllSupportedFeatures: *flags.EnableAllSupportedFeatures,
-		},
-		Implementation: v1alpha1.Implementation{
+	opts := conformance.DefaultOptions(t)
```

Comment:
Does this now just grab the options from the flags?

Reply:
Yep, it does everything I've removed here. They made a change [here](https://github.com/kubernetes-sigs/gateway-api/pull/2864) which caused a panic when we weren't using default options - there apparently was a change made after which defaulted the manifest file system value to avoid a panic, but I was still getting the panic in my testing. Anyway, this seems to work well for us, and brought in other fixes (like adding in "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1") automatically.